pkg/installation/move.go

```diff
@@ -165,6 +165,11 @@ func moveToInstallDir(srcDir, installDir string, fos []index.FileOperation) erro
 	}
 	defer os.RemoveAll(tmp)
 
+	if fos == nil {
+		fos = []index.FileOperation{{From: "*", To: "."}}
```

Comment:
this makes the method less testable :|

semantically, if fos==nil you'd expect this method to do nothing, based on its current name + godoc. :)